_src/doxygen/doxygen.css

```diff
@@ -658,31 +664,28 @@ td.memItemLeft div.content, td.memTemplItemLeft div.content {
 }
 
 .memproto, dl.reflist dt {
-        padding: 6px 0px 6px 0px;
-        color: #ddd;
-        font-weight: bold;
-        background-color: #202020;
-        /* opera specific markup */
-        box-shadow: 5px 5px 5px rgba(0, 0, 0, 0.15);
-        border-top-right-radius: 4px;
-        /* firefox specific markup */
-        -moz-box-shadow: rgba(0, 0, 0, 0.15) 5px 5px 5px;
-        -moz-border-radius-topright: 4px;
-        /* webkit specific markup */
-        -webkit-box-shadow: 5px 5px 5px rgba(0, 0, 0, 0.15);
-        -webkit-border-top-right-radius: 4px;
-
+	padding: 6px 0px 6px 0px;
+	color: #ddd;
+	font-weight: bold;
+	/* opera specific markup */
+	box-shadow: 5px 5px 5px rgba(0, 0, 0, 0.15);
+	border-top-right-radius: 4px;
+	/* firefox specific markup */
+	-moz-box-shadow: rgba(0, 0, 0, 0.15) 5px 5px 5px;
+	-moz-border-radius-topright: 4px;
+	/* webkit specific markup */
+	-webkit-box-shadow: 5px 5px 5px rgba(0, 0, 0, 0.15);
+	-webkit-border-top-right-radius: 4px;
 }
 
 .overload {
-        font-family: "courier new",courier,monospace;
+    font-family: "courier new",courier,monospace;
```

Comment:
Hmm, should we use `Roboto` here?  Looks like I might have overlooked that in the past.

Reply:
Done, but we should see how it looks like once you rebuild doxygen.